modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/application/endpoint/EndpointMatcher.java

```diff
@@ -74,12 +75,12 @@ private String _getSingleElementRegex(APIApplication.Endpoint endpoint) {
 
 		if (Objects.equals(
 				endpoint.getPathParameter(),
-				APIApplication.Endpoint.PathParameter.ID)) {
+				HeadlessBuilderConstants.PATH_PARAMETER_ID)) {
 
 			return endpointPath + "\\d+";
```

Comment:
I think we talked about removing this validation :thinking:

Reply:
As for the numeric unique fields, we would not be including this one and it would use the other one, so it does not make sense to have two possible validations for numberic fields

Reply:
Yeah, I think that finally we are going to accept all, instead of the slash